### ISSUE TITLE
Add Your Own Custom Name!

### ISSUE THREAD

**Issue by account21** [leomwu/agario-bot#11]
This isn't as much of an issue as it is a cool thing to try out.
If you edit the main out js file and scroll to the way bottom you can add, remove, and change names without any coding knowledge at all!
I have a suggestion though. Could there be a way to possibly be able to type in your name like you normally would on agar.io instead of it giving you predetermined names? This would not be good for the people that just want to let the computer do it's thing but maybe you could have a box that you could tick for that option?
Awesome bot, by the way!

**Comment by minecrafter1094** [comment]
This is a feature I would like to see. Maybe in the unpacked extension folder you download there could be a names.json file, I don't know what file is easily read for you (maybe .csv?) but otherwise, yes.

**Comment by LeoWuVinci** [comment]
This would be something I would consider when the framework is more popular :)

**Comment by phadeb** [comment]
I endorse this, this shouldn't be hard or time consuming to code.

**Comment by cj81499** [comment]
Leo didn't want to let users choose their own names because he felt they shouldn't get credit for the bot's work.

EDIT: I know because I asked him about it on the stream a while back.

**Comment by jononon** [comment]
I agree. Lots of people are going to be using this bot and Lio deserves credit for his work. If you want your own name, just fork the repo.

**Comment by cj81499** [comment]
Pretty much exactly how I feel.

**Comment by jononon** [comment]
Names are stored at line 2115 here: https://github.com/leomwu/agario-bot/blob/master/main_out_519v2.js

```
window.skinNames=[
    'twitch/gamerlio',
    'nomday.com/lio',
    'Yaranaika',
    'Pokerface',
    'Sir',
    'Mars',
    'Stalin',
    'Moon',
    'Wojak',
    'Imperial Japan',
    'Doge',
    'Earth',
    'Bait',
    'Steam',
    'Piccolo',
    'Sanik',
    'Cia',
    '4chan',
    'Ayy Lmao',
    'Qing Dynasty',
]
```

Fork and change.
